server/lib/s3backup/__init__.py

```diff
@@ -64,6 +63,11 @@ def __init__(self, config, logger):
         else:
             self.connector = S3Connector(config)
 
+        try:
+            self.bucket_name = self.connector.bucket_name
+        except Exception:
+            pass
```

Comment:
Can we add a comment explaining why we can `pass` here and ignore the exception?

Reply:
Yes, I'll do that.